test/jdk/java/lang/Float/Binary16ConversionNaN.java

```diff
@@ -25,6 +25,9 @@
  * @test
  * @bug 8289551
  * @summary Verify NaN sign and significand bits are preserved across conversions
+ * @run main Binary16ConversionNaN
+ * @run main/othervm/timeout=600 -XX:+UnlockDiagnosticVMOptions
```

Comment:
Do we need to specify a timeout? The tests have been running fine without intrinsics, no timeouts reports.